### DISCUSSION TITLE
New organ : problem on loading

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#697] [General]
Hello,

I composed an organ with some different organs available for GrandOrgue and Hauptwerk.
I used as usual the Vincent Forman's Organ builder program.

When my organ was finished, i tested it from Vincent Forman's prrogram.
Everything was ok.

But when i load it on GrandOrgue (version v0.3.1.2341-15.os), i have this message

> 16/09/2021 12:41:58: 12:41:58: Erreur : Valeur hors limite section 'Organ' entrée 'NumberOfRanks': 443

I don't understand what happened and don't know how to correct that ..
Have you an idea ?

Many thanks :)

**Comment by oleg68** [comment]
This topic is a duplicate of #680 so I lock this.
